lib/reporter.js

```diff
@@ -325,12 +325,27 @@ class Reporter {
             .promise.catch(errorHandler);
     }
 
+    finishUndefinedStep() {
+        for (const item of this.collectionMap.values()) {
+            for (const step of item.steps) {
+                if (step.status === undefined) {
+                    let finishTestItemRQ = {
+                        endTime: new Date().valueOf(),
+                        status: 'SKIPPED'
```

Comment:
Please use status as constant defined at `TestStatus`

Reply:
Oh, thanks! I just fixed this